pvc_test.go

```diff
@@ -0,0 +1,147 @@
+package sonobuoy_plugin
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/sonobuoy-plugin/v5/pkg/ctrlclient"
+)
+
+const (
+	pvcName      = "mypvc"
+	pvcNamespace = "default"
+)
+
+// Test_PVC tests PVCs with default storage class are being provisioned.
+func Test_PVC(t *testing.T) {
+	t.Parallel()
+
+	var err error
+
+	ctx := context.Background()
+
+	tcCtrlClient, err := ctrlclient.CreateTCCtrlClient()
+	if err != nil {
+		t.Fatalf("error creating CP k8s client: %v", err)
+	}
+
+	regularLogger, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	logger := NewTestLogger(regularLogger, t)
+
+	pvc, err := createPVC(ctx, tcCtrlClient)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pod, err := createPod(ctx, tcCtrlClient)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Cleanup(func() {
+		_ = tcCtrlClient.Delete(ctx, pvc)
+		_ = tcCtrlClient.Delete(ctx, pod)
```

Comment:
does the order matter? I was wondering if `pod` should be deleted first

Reply:
I tested it and it just works so I guess it's not important